baggysoapshop/urls.py

```diff
@@ -29,7 +29,7 @@
 
     path('i18n/', include('django.conf.urls.i18n')),
     path('checkout/paypal/', include('paypal.express.urls')),
-
+    path('dashboard/paypal/express/', apps.get_app_config("express_dashboard").urls),
```

Comment:
Great to have this working, finally! Even if the information within it seems rather useless... :laughing: Perhaps you could delete the comment below?